scripts/speech_recognition/convert_to_tarred_audio_dataset.py

```diff
@@ -62,10 +68,21 @@
     --max_duration=16.7 \
     --min_duration=0.01 \
     --shuffle \
-    --sort_in_shards
+    --workers=-1 \
+    --sort_in_shards \
     --shuffle_seed=1 \
     --write_metadata
 
+
+python convert_to_tarred_audio_dataset.py \
+    --manifest_path="/home/smajumdar/PycharmProjects/nemo-eval/nemo_beta_eval/librispeech/manifests/dev_test_all.json" \
+    --target_dir="/media/smajumdar/data/Datasets/Librispeech_Dev_Test_Tarred/" \
```

Comment:
I would replace it with a generic place holder.

Reply:
Ah my bad. Haha this should not be part of the PR but for my testing only.

Reply:
👍